src/App.tsx

```diff
@@ -1,5 +1,29 @@
+import colors from './colors.json';
+
+const ColorsGreeting = ({ name, hex, comp }) => {
+    return (
+        <div>
+            <h1 style={{ color: `#${hex}`, padding: '2em 2em 0em 2em', fontSize: '2em', display: 'flex', justifyContent: 'center' }}>Hello human, I am {name}.</h1>
+            <p style={{ color: 'white', fontSize: '1.2em', display: 'flex', justifyContent: 'center' }}>My best friends are:</p>
+            <ul style={{ color: `#${hex}`, fontSize: '1.2em', display: 'flex', justifyContent: 'center' }}>
+                {comp.map(color => (
+                    <li key={color.name} style={{ color: `#${color.hex}`, marginRight: '1em' }}>
+                        {color.name}
+                    </li>
+                ))}
+            </ul>
+        </div>
+    );
+};
+
 const App = () => {
-    return <h1>Hello World</h1>;
+    return (
+        <div>
+            {colors.map(color => (
+                <ColorsGreeting key={color.name} name={color.name} hex={color.hex} comp={color.comp} />
```

Comment:
Using the color name for the key is likely fine in this case but in general, try to use something that will definitely be unique, like an index or the hex code